FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/Models/DicomToDataLakeProcessingJobResult.cs

```diff
@@ -0,0 +1,48 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Jobs.Models
+{
+    public class DicomToDataLakeProcessingJobResult
```

Comment:
Is this one exactly the same with the one for FHIR? If so, can we make a common class for both data sources?

Reply:
No, it is different. we don't need patient version information in `DicomToDataLakeProcessingJobResult`

Reply:
Got it. Thanks!